Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -197,6 +197,15 @@ void WebProcessProxy::forWebPagesWithOrigin(PAL::SessionID sessionID, const Secu
     }
 }
 
+void WebProcessProxy::forWebPagesWithDataStore(const WebsiteDataStore& websiteDataStore, const Function<void(WebPageProxy&)>& callback)
```

Comment:
This can be folded into `WebProcessProxy::isBlobRegistryPartitioningEnabled(SessionID)`.